### PR TITLE
[#258] #IMPLEMENT 'assemblyName: DotNet.Testcontainers; function: CouchbaseTestcontainer' {Add pre-configured Couchbase Testcontainer.}

### PR DESCRIPTION
**Pull request by mustafaonuraydin**
[#258] #IMPLEMENT 'assemblyName: DotNet.Testcontainers; function: CouchbaseTestcontainer' {Add pre-configured Couchbase Testcontainer.}

### CODE REVIEW

**Comment by mustafaonuraydin** [comment]
@HofmeisterAn I create new pull request with new branch. I tried to refactor and improve with you suggestion. Additionally, i updated latest version of Couchbase.
You can see customized docker image source from following repository;
https://github.com/mustafaonuraydin/couchbase-docker-image-for-testcontainers

After your review please feel free to ask for other changes or ideas and i really appreciate to discuss over code.

File: src/DotNet.Testcontainers/Containers/Configurations/Databases/CouchbaseTestcontainerConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+{
+  using System;
+  using System.IO;
+  using Abstractions;
+  using OutputConsumers;
+  using WaitStrategies;
+
+  public sealed class CouchbaseTestcontainerConfiguration : TestcontainerDatabaseConfiguration
+  {
+    private const string CouchbaseImage = "mustafaonuraydin/couchbase-testcontainer:6.5.1";
```
Comment (HofmeisterAn) [inline src/DotNet.Testcontainers/Containers/Configurations/Databases/CouchbaseTestcontainerConfiguration.cs:39]: Maybe you can add some information (comment) about the image here until the sources are public.

**Review by HofmeisterAn — CHANGES_REQUESTED** [review]
LGTM 👍 It would be nice if you could add the suggested comment below. Then I'll merge it later that day.

File: src/DotNet.Testcontainers/Containers/Configurations/Databases/CouchbaseTestcontainerConfiguration.cs (hunk 1 above)
Comment (mustafaonuraydin) [reply src/DotNet.Testcontainers/Containers/Configurations/Databases/CouchbaseTestcontainerConfiguration.cs:39]: Coming 👍

**Comment by mustafaonuraydin** [comment]
@HofmeisterAn i sent updates with your suggestion. Thanks for your support! I hope this feature will helpful to community. 🎉

**Review by HofmeisterAn — APPROVED** [review]

**Comment by HofmeisterAn** [comment]
Merged with 2cc3c6e398f9e6b068f0f4a306888fd251502a70 and c78c30bf940d9a8812c5c128ab89f777040b0974.

**Pull request closed without merging**